dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsApiExtendedServiceImpl.java

```diff
@@ -197,41 +197,25 @@ public Response toolsIndexGet(SecurityContext securityContext) {
         return Response.ok().entity(0).build();
     }
 
+    @Override
+    public Response entriesIndexSearch(String query, MultivaluedMap<String, String> queryParameters, SecurityContext securityContext) {
+        final String index = "tools,workflows";
```

Comment:
This is minor, but since we someday might have dedicated indexes for services and/or notebooks, I would move this up to the top along with constants for `tools` and `workflows`. Actually it looks like you also have the index names in ElasticListener -- the index names should be defined once, ideally.